churaverse-plugins-client/src/churarenPlugin/churarenBossPlugin/store/initBossPluginStore.ts

```diff
@@ -0,0 +1,25 @@
+import { IMainScene, Store } from 'churaverse-engine-client'
+import { IBossRendererFactory } from '../domain/IBossRendererFactory'
+import { BossPluginStore } from './defBossPluginStore'
+import { BossRepository } from '../repository/bossRepository'
+import { IBossRenderer } from '../domain/IBossRenderer'
+import { DamageCauseLogRenderer } from '../ui/damageCauseLog/damageCauseLogRenderer'
+import { DamageCauseLogRepository } from '../ui/damageCauseLog/deathLogRepository'
+
+export function initBossPluginStore(store: Store<IMainScene>, rendererFactory: IBossRendererFactory | undefined): void {
+  if (rendererFactory === undefined) throw Error('rendererFactory is undefined')
+
+  const bossPluginStore: BossPluginStore = {
+    bosses: new BossRepository(),
+    bossRenderers: new Map<string, IBossRenderer>(),
+    bossRendererFactory: rendererFactory,
+    damageCauseLogRenderer: new DamageCauseLogRenderer(store.of('coreUiPlugin').fadeOutLogRenderer),
+    damageCauseLogRepository: new DamageCauseLogRepository(),
+  }
+
+  store.setInit('bossPlugin', bossPluginStore)
+}
+
+export function resetBossPluginStore(store: Store<IMainScene>): void {
+  store.deleteStoreOf('bossPlugin')
+}
```

Comment:
ファイルの最後は改行を入れるようにした方が良いですね（おそらくVSCodeのフォーマッタが効いていれば、保存時に勝手に付け加えられる気はします）
参考：https://qiita.com/hamacccccchan/items/11c17c7412a5aeb2ad74

```suggestion
}

```